server/src/main/java/org/elasticsearch/cluster/metadata/MetadataCreateIndexService.java

```diff
@@ -623,7 +636,22 @@ public ClusterState add(ClusterState currentState,
         // create the index here (on the master) to validate it can be created, as well as adding the mapping
         return indicesService.withTempIndexService(tmpImd, indexService -> {
             IndexAnalyzers indexAnalyzers = indexService.indexAnalyzers();
-            ensureUsedAnalyzersExist(indexAnalyzers, request.references());
+            ensureUsedAnalyzersExist(indexAnalyzers, table.columns());
+            metadataBuilder.setTable(
+                table.name(),
```

Comment:
Why is this setting the table again? That happens already in:

https://github.com/crate/crate/blob/31d9ee296cefa5de880eb4381a4ee0d03285f72a/server/src/main/java/io/crate/execution/ddl/tables/TransportCreateTable.java#L172-L186